apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/IgnoredTracerContext.java

```diff
@@ -53,7 +53,7 @@ public void extract(ContextCarrier carrier) {
 
     @Override
     public ContextSnapshot capture() {
-        return new ContextSnapshot(null, -1, null, null, correlationContext);
+        return new ContextSnapshot(null, -1, null, null, correlationContext, new ExtensionContext());
```

Comment:
`#capture` should propogate the `ExtensionContext`, rather than creating a new one.

Reply:
I can see it is inside the tracing context.

Reply:
I create a new field to save the extension context in the ignored context.